ts/components/wallet/PaymentMethodsList.tsx

```diff
@@ -153,9 +160,9 @@ class PaymentMethodsList extends React.Component<Props, never> {
           }}
         />
         <View spacer={true} large={true} />
-        <Text link={true} onPress={this.showHelp}>
-          {I18n.t("wallet.whyAFee.title")}
-        </Text>
+        <TouchableWithoutOpacity onPress={this.showHelp}>
+          <Text style={styles.linkStyle}>{I18n.t("wallet.whyAFee.title")}</Text>
```

Comment:
Same ad above: style is not needed
![original](https://user-images.githubusercontent.com/822471/68234411-710f0b00-0001-11ea-9172-8c8dbbd9e768.png)